app/code/Maksym/CustomCategory/view/frontend/templates/subcategory.phtml

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**@var Maksym/CustomCategory/ViewModel/Product/SubCategory $viewModel*/
+$viewModel = $block->getData('view_model'); ?>
+
+<div class="subcategory" data-mage-init='{
+        "accordion":{
+            "active": "false",
+            "collapsible": true,
+            "openedState": "deactivate",
+            "multipleCollapsible": true
+        }}'>
+    <div class="filter-options-wrapper">
+        <div class="filter-options-title" data-role="title">Category</div>
+        <div class="filter-options-content" data-role="collapsible">
+            <ol class="items" data-role="content">
+                <?php
+        $subcats = $viewModel->getSubcategories();
+        foreach ($subcats as $subcat) {
+            if ($subcat->getIsActive()) {
+                $subcat_url = $subcat->getUrl();
+        ?>
+                <li class="item">
+                    <a class="sub-category__link" href="<?php echo $subcat_url; ?>">
+                        <p><?php echo $subcat->getName(); ?></p>
```

Comment:
use $escaper for `$subcat->getName()` and `$subcat_url`

Reply:
done